### ISSUE TITLE
Manage User Cache Problems

### ISSUE THREAD

**Issue by aarontitus** [CrisisCleanup/crisiscleanup-3-web#1288]
**Describe the bug**
When editing users through https://www.crisiscleanup.org/organization/users, the View Roles version of the user profile and the Edit Profile version contain different roles: Probably some sort of caching issue. See screenshots.

**Screenshots**
![user_17568 Roles](https://user-images.githubusercontent.com/4184619/131287176-d0063228-c64d-4e76-a120-55c4b0ff1a6d.png)
![user_17568 Roles edit modal](https://user-images.githubusercontent.com/4184619/131287187-b9c96231-ea12-47c2-a189-746e77973df4.png)

**Comment by BradenM** [comment]
@aarontitus,

![image](https://user-images.githubusercontent.com/5913808/131332281-c84afe67-6066-4d15-a5e4-4827b5937d45.png)
![image](https://user-images.githubusercontent.com/5913808/131332289-c392e6d4-a926-428d-948b-eb7dfaca19ee.png)

Am I crazy and/or just blatantly missing something or do these two images not show the same thing? :thinking:

**Comment by aarontitus** [comment]
@BradenM, nope, you're not crazy, but I happened to grab the wrong example. I promise that there is a difference between the two in other circumstances.
